packages/account/src/Containers/reset-trading-password.tsx

```diff
@@ -36,7 +37,7 @@ const ResetTradingPassword = observer(() => {
             enableApp={enableApp}
             disableApp={disableApp}
             toggleResetTradingPasswordModal={setResetTradingPasswordModalOpen}
```

Comment:
why we didnt changes the toggleFunction to " setCFDPasswordResetModal"

Reply:
sure
will refactor the action and observer... it is used by multiple packages....

Reply:
@amina-deriv can I address these in a separate card? because the changed file count will greatly increase, since this change will involve 9+ file more

Reply:
i think without changing all place, it will effect tradersHub and accounts( without wallet). can u double check

Reply:
@shontzu-deriv let's change it. We need to make sure that all flows will work as expected.

Reply:
done!